Code/GraphMol/Conformer.cpp

```diff
@@ -48,8 +48,8 @@ const RDGeom::POINT3D_VECT &Conformer::getPositions() const {
   return d_positions;
 }
 
-RDGeom::POINT3D_VECT &Conformer::getPositions() {
```

Comment:
Had to add a force function to get positions after atoms were removed.  Do we really need to check the position size here, I'm curious what this buys us or if it just a helper for molecule modification.

Reply:
Not happy with my name/api btw,.

Reply:
instead of changing the public API, I would just get rid of the check entirely

Reply:
Reverted